auth-web/src/components/auth/manage-business/EntityManagement.vue

```diff
@@ -581,15 +581,19 @@ export default class EntityManagement extends Mixins(AccountMixin, AccountChange
       try {
         await this.syncOrganization(this.currentAccountSettings.id)
         await this.addOrgSettings(this.currentOrganization)
+        this.isLoading = 1 // truthy
```

Comment:
Why not just use a boolean?

Reply:
it used !!isLoading I think, not too sure

Reply:
!!isLoading 

![image](https://github.com/bcgov/sbc-auth/assets/3484109/c9844b2a-7ad5-400c-b2bc-335ad4d5c41c)